### PR TITLE
Add integration tests #2 for `percy-node-client.js`

### PR DESCRIPTION
**Pull request by bogusred**
Add integration tests #2 for `percy-node-client.js`

- Revised `percy-node-client.js` to use a separate logger method so it can be stubbed.
- added jasmine tests. Decided on integration instead of unit for now since this offers more confidence as to whether it works as intended. Might add unit tests later.

### CODE REVIEW

**Comment by bogusred** [comment]
Please choose `rebase and merge` option to get clean history.

**Comment by bogusred** [comment]
scratch that. please choose squash and merge. rebase and merge does not squash. oops

File: test/jasmine.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "spec_dir": "test",
+  "spec_files": [
+    "**/*[sS]pec.js"
```
Comment (humidair1999) [inline test/jasmine.json]: Can we keep this just lowercase to ensure filename consistency?

**Review by humidair1999 — APPROVED** [review]

File: test/jasmine.json (hunk 1 above)
Comment (bogusred) [reply test/jasmine.json]: Good point. thank you i'll update.

**Pull request merged**